src/components/breadcrumb-bar/breadcrumb-bar.module.css

```diff
@@ -26,10 +33,32 @@
 }
 
 .breadcrumbText {
-  color: var(--token-color-palette-neutral-500);
+  composes: g-focus-ring-from-box-shadow from global;
```

Comment:
@ashleemboyer Update since last review: these are the updates for breadcrumb link styles.